packages/theme/src/index.js

```diff
@@ -1,4 +1,4 @@
-import createTheme from './createTheme';
+import createTheme from './createTheme'; // deprecated
```

Comment:
Quick Question
Do we need to add the `// deprecated` comment here?

Reply:
The `createTheme` function exported from the `theme` package may be removed in the next major release to prevent confusion. I’m adding this comment here as a deprecation notice.

Reply:
See https://github.com/trendmicro-frontend/tonic-ui/issues/892

### `@tonic-ui/theme`

* The `createTheme` function has been removed from the named exports.
  ```js
  // Unit-specific themes
  export const rem = _rem(foundation);
  export const px = _px(foundation);

  // Export foundation directly
  export { foundation };
  ```